integration/spark/src/test/common/java/io/openlineage/spark/agent/SparkContainerIntegrationTest.java

```diff
@@ -333,6 +333,30 @@ public void testOptimizedCreateAsSelectAndLoad() {
     verifyEvents("pysparkOCTASStart.json", "pysparkOCTASEnd.json");
   }
 
+  @Test
+  @EnabledIfSystemProperty(named = "spark.version", matches = SPARK_3) // Spark version >= 3.*
```

Comment:
This can be v2 test, but for `input` Delta uses LogicalRelation/HadoopFsRelation with v1 CatalogTable.

After input is done, this will need to be a separate, non-v2 test as well.